### PR TITLE
Rename autoplay_enabled to tts_autoplay_enabled

### PR DESCRIPTION
**Pull request by maureensturgeon**
Migration to update `autoplay_enabled` to `tts_autoplay_enabled` for clarity (motivation: https://github.com/code-dot-org/code-dot-org/pull/34700#discussion_r468080713).

These changes were pulled from https://github.com/code-dot-org/code-dot-org/pull/36409

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by jamescodeorg — APPROVED** [review]
Looks good to me although I'm not confident that I know all the things I should be looking for.  Might be good to get a second opinion from someone who's more familiar with db migrations.

**Review by dmcavoy — APPROVED** [review]
This looks good to me. Unfortunately I'm not much more experienced with rail migration to be able to confidently say if there are concerns. Some thing I'm wondering about is if these tables are big enough where this change might take some time. That might be a good question to ask Suresh.

**Comment by sureshc** [comment]
There are about 3 million rows in the production Section table, so it's worth investigating whether applying this migration will negatively impact database performance.

Looks like [when MySQL 5.7 renames a column](https://dev.mysql.com/doc/refman/5.7/en/innodb-online-ddl-operations.html), it only needs to update metadata about the table and does NOT need to rebuild the entire table "in place" so this migration is safe to run on production.

image

**Review by sureshc — APPROVED** [review]

File: dashboard/app/models/sections/section.rb [hunk 1]
```diff
@@ -256,7 +256,7 @@ def summarize(include_students: true)
       code: code,
       lesson_extras: lesson_extras,
       pairing_allowed: pairing_allowed,
-      autoplay_enabled: autoplay_enabled,
+      tts_autoplay_enabled: tts_autoplay_enabled,
```
Comment (dmcavoy) [inline dashboard/app/models/sections/section.rb:259]: This should have probably been:

```
autoplay_enabled: tts_autoplay_enabled,
tts_autoplay_enabled: tts_autoplay_enabled

```

**Pull request merged**